### DISCUSSION TITLE
workspace tag not working as expected as codebase context

### DISCUSSION THREAD

**Discussion by mcarturo** [community/community#77427] [Copilot Conversations]
### Select Topic Area

Question

### Body

Hi,

I have installed the latest version of github copilot chat in vscode. Whenever I use the @workspace tag I expected to be able to ask questions about my current vscode workspace but instead I am getting the following message from the assistant:

`As an AI, I don't have direct access to your file system or IDE, so I can't`

Is this the expected behaviour? Is there a way I can troubleshoot this?

Thanks¡¡

I am using this version of copilot and vscode:
- Version: 1.139.575
- Editor: vscode/1.84.2

on ubuntu and running the chat within an nx monorepo.

**Comment by Akash1134** [comment]
Hi @mcarturo ,

I have shared this with my team internally to look into it. Meanwhile, I'd recommend that you review these [release notes](https://code.visualstudio.com/updates/v1_84#_chat-agents) to ensure you are using the Agent as you should be. 

Thenks!

**Reply by mcarturo** [reply]
Hello @Akash1134 , thanks for your reply, 

I have been working some days with copilot and I have realized that I was not using it the way I should (for the time being one cannot expect answers to general questions about the whole project codebase :) ).

It worked as expected.

Thanks again.
